src/lib/stateBasedTestHelper.ts

```diff
@@ -201,7 +201,7 @@ export async function testSagaWithState<StateT, Payload>(
         return funcOrBoundEffect(testContext, ...args);
       }
     },
-    take: (actionCreator) => {
+    take: (actionCreator: ActionCreator<any>) => {
```

Comment:
I guess we don't need the timeout here in practice, because the test runner will err due to timeout anyway :)

Reply:
Yeah, not sure what we should do in tests with `take`.